samples/subsys/usb/cdc_acm_echo/src/main.c

```diff
@@ -0,0 +1,122 @@
+/*
```

Comment:
:rocket: An other CDC ACM sampel :smile: Is it possible to integrate it to `subsys/usb/cdc_acm/src/main.c` and select|configure over `subsys/usb/cdc_acm/Kconfig`?

Reply:
What if I remove `cdc_acm` and leave only this sample. If I use config and `ifdefs` the code would be unreadable.

Reply:
Why not rework the code of cdc_acm sample?